client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/integration/web/BioWebClient.kt

```diff
@@ -26,7 +27,7 @@ class BioWebClient internal constructor(
             SubmissionClientImpl(
                 createRestTemplate(baseUrl, token, onBehalf),
                 SerializationConfig.serializationService(),
-                ExtSerializationService()
+                ExtSerializationConfig.extSerializationService(baseUrl)
```

Comment:
lets make it relative to avoid passing this baseUrl to this element

Reply:
done